ServerCore/Pages/Puzzles/Checklist.cshtml

```diff
@@ -61,7 +61,14 @@
                 <a asp-Page="./Edit" asp-route-puzzleid=@item.Puzzle.ID>@Html.DisplayFor(modelItem => item.Puzzle.Name)</a>
             </td>
             <td>
-                @Html.DisplayFor(modelItem => item.Puzzle.Group)
+                @if (item.Puzzle.Group != null && item.Puzzle.Group.StartsWith("Html.Raw(") && item.Puzzle.Group.EndsWith(")"))
```

Comment:
Why would group names be html?

Reply:
PH 22 asked for it, that's all I know. I guess we'll find out in September!